### DISCUSSION TITLE
SHUTTING IT DOWN! microsoft style 😎

### DISCUSSION THREAD

**Discussion by CSTEZCAN** [microsoft/AirSim#4627] [General]
they are using all of our feedback to create the best simulator and shutting it down. 

They blocked my account and removed my comments under the post shortly after. 

Do not let them get away with this. 

![Screenshot 2022-07-18 at 13-29-47 Post Feed LinkedIn copy](https://user-images.githubusercontent.com/33690601/179500244-61c1a41e-28e7-4238-a483-72f0aee668a1.png)

**Comment by salon2525** [comment]
My personal suggestion would be that we as a community make a fork of it & then contribute to its further development. We will need funds for that, may be you can setup something to ask for donation & we can create a new open source version of airsim? 

Microsoft is free to do what they believe is best for their interest, Right?

**Comment by xxEoD2242** [comment]
My company, [Codex Laboratories LLC](https://www.codex-labs-llc.com), is looking to continue and partially fund the continued open source project. We have a platform that was based on AirSim, the [SWARM Simulation Platform](https://www.swarmsim.io), which we subsequently heavily modified and will continue development of AirSim, porting back important changes as open source when we can.

We are also in conversation with Purdue University to provide some nominal support as well and have maintainers/contributors. If anyone is interested, please reach out to me directly: Tyler Fedrizzi (tylerfedrizzi@codex-labs-llc.com)

**Comment by xxEoD2242** [comment]
Here is a link to the fork: [Colosseum, successor to AirSim](https://github.com/CodexLabsLLC/Colosseum)
